Cargo.toml

```diff
@@ -12,7 +12,6 @@ autobenches = false
 
 [features]
 default = []
-integer-atomics = []
```

Comment:
This seems like a breaking change, should we bump the minor version number for next release?